release/models/bgp/openconfig-bgp-neighbor.yang

```diff
@@ -305,6 +312,23 @@ submodule openconfig-bgp-neighbor {
         due to an inbound connection request from a specified source prefix
         within a dynamic-neighbor-prefix.";
     }
+
+    leaf last-prefix-limit-exceeded {
+      type oc-types:timeticks64;
+      description
+        "This timestamp indicates the time that the BGP session last
+        violated a configured max prefix-limit for any AFI/SAFI combination
+        on the session.  The value is the timestamp in nanoseconds relative
+        to the Unix Epoch (Jan 1, 1970 00:00:00 UTC).
+
+        If the 'prefix-limit-exceeded' node is set to true for any AFI/SAFI
+        on the session, then the next restart time for the session can be
+        calculated as this value plus the configured 'restart-time' under
+        the neighbor timers.
+
+        This value should be retained across established sessions and is only
+        set/updated during a prefix-limit-exceeded event.";
```

Comment:
Possibly even more clear? 

```suggestion
        set/updated when prefix-limit-exceeded transitions from false to true.";
```

Reply:
ack thx - addressed in https://github.com/openconfig/public/pull/532/commits/7ce1d279d92c56dfdaa29640e8c4d82d3e04d0ac